docs/content/docs/deployment/advanced/logging.md

```diff
@@ -67,6 +67,8 @@ The Flink distribution ships with the following log4j properties files in the `c
 Log4j periodically scans this file for changes and adjusts the logging behavior if necessary.
 By default this check happens every 30 seconds and is controlled by the `monitorInterval` setting in the Log4j properties files.
 
+You can configure the default logging level using the [`env.log.level` configuration]({{< ref "docs/deployment/config">}}#jvm-and-logging-options).
```

Comment:
The sentence at the top says "Log4j 2 is controlled using property files." This is not correct - we add and also the configuration.

Maybe have 2 subsections one for the properties and one for the configuration. I suggest documenting the other 2 log related configurations options also here.  

I think is is confusing using the word default - as this setting has a default. I think it would be easier to parse using words like : you can configure the logging level max and dir in the configuration . The logging level specified in the configuration can be overridden in the log4j property files.

Reply:
Good points. Updated.